lib/html-proofer/check/links.rb

```diff
@@ -49,7 +49,7 @@ def run
         next if @link.respond_to?(:rel) && @link.rel == 'dns-prefetch'
         add_to_external_urls(@link.href)
         next
-      elsif !@link.internal? && !@link.exists?
+      elsif !@link.remote? && !@link.exists?
```

Comment:
Wouldn't this be best as `@link.internal?` (no `!`)?

Reply:
It would. I tried it and it created some more errors failing. Looks to be some discrepancy there. I will give another go.